indico/modules/events/contributions/controllers.py

```diff
@@ -206,3 +216,67 @@ def _get_defaults(self):
         acl = {p.principal for p in self.contrib.acl_entries if p.read_access}
         return {'managers': managers, 'submitters': submitters, 'protection_mode': self.contrib.protection_mode,
                 'acl': acl}
+
+
+def _contribution_start_date_form(contrib):
+    class _ContributionStartDateForm(IndicoForm):
+        def __init__(self, *args, **kwargs):
+            self.contrib = kwargs.pop('contrib')
+            self.conf = self.contrib.event_new.as_legacy
+            self.timezone = self.conf.getTimezone()
+            super(_ContributionStartDateForm, self).__init__(*args, **kwargs)
+
+        @property
+        def start_dt(self):
+            return getattr(self, 'start_dt-{0}'.format(self.contrib.id))
```

Comment:
Otherwise `datepicker` will not work as it should, since every contribution row in the table will have a datepicker with the same ID, thus I need to give every datepicker a separate id..